ephios/core/templatetags/user_extras.py

```diff
@@ -18,6 +18,12 @@ def editable_consequences_tag(user, states=None):
     return qs
 
 
+@register.filter(name="pending_consequences")
+def pending_consequences_filter(user):
```

Comment:
```suggestion
register.filter(name="pending_consequences")(pending_consequences)
```

Reply:
should suffice?! (no need to build an id function)
Maybe there also is nicer syntax to register existing syntax as filters.